OpenRA.Mods.Cnc/Traits/Buildings/ClonesProducedUnits.cs

```diff
@@ -39,7 +39,7 @@ public ClonesProducedUnits(ActorInitializer init, ClonesProducedUnitsInfo info)
 			faction = init.Contains<FactionInit>() ? init.Get<FactionInit, string>() : init.Self.Owner.Faction.InternalName;
 		}
 
-		public void UnitProducedByOther(Actor self, Actor producer, Actor produced)
+		public void UnitProducedByOther(Actor self, Actor producer, Actor produced, string type)
```

Comment:
These are actually the production queue `Type`s, which makes sense in this context.  It would make sense to rename `type` and `exitType` to `productionType` and `ExitInfo.Types` to `ExitInfo.ProductionTypes` to make this clearer.